libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationService.java

```diff
@@ -170,6 +157,7 @@ void acquireLocationEngine() {
     locationEngine = mapboxNavigation.getLocationEngine();
     locationEngine.addLocationEngineListener(this);
     locationEngineName = obtainLocationEngineName();
```

Comment:
If possible, let's remove the field `locationEngineName` and use `obtainLocationEngineName()` when updating the queue class.